verifiable-db/src/query/universal_circuit/universal_query_circuit.rs

```diff
@@ -173,7 +174,9 @@ where
             .chain(empty_hash.elements.iter())
             .chain(node_min.to_targets().iter())
             .chain(node_max.to_targets().iter())
-            .chain(once(second_index_id))
+            .chain(once(
+                &hash_wires.input_wires.column_extraction_wires.column_ids[1],
```

Comment:
why are we adding this one again ?

Reply:
Not sure what you mean. You are wondering why I didn't keep the `second_index_id` variable?